Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -21,13 +21,11 @@ class Navigator {
     static var tilesURL: URL? = nil
     
     /**
-     Path to the directory where history file could be stored when `Navigator.writeHistory(completionHandler:)` is called. Defaults to user support directory.
+     Path to the directory where history file could be stored when `Navigator.writeHistory(completionHandler:)` is called.
+     
+     Setting `nil` disables history recording. Defaults to `nil`.
      */
-    static var historyDirectoryURL: URL = {
-        let supportDir = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first!
-        
-        return supportDir.appendingPathComponent("com.mapbox.navigation.history")
-    }()
+    static var historyDirectoryURL: URL? = nil
```

Comment:
Would it be possible to write a unit-test for this case? E.g. when `historyDirectoryURL` is `nil` `writeHistory(completionHandler:)` should not dump any data, right?

Reply:
Added tests